### PR TITLE
Increase responsiveness of osu! logo triangles to the beat

### PR DESCRIPTION
**Pull request by TacoGuyAT**
Logo triangles are now dashing on each beat and their speed is tweaked

https://user-images.githubusercontent.com/35270005/180124238-1ec05a17-b22b-4154-a76a-3009165a2e89.mp4

### CODE REVIEW

File: osu.Game/Graphics/Containers/BeatSyncedContainer.cs [hunk 1]
```diff
@@ -28,7 +28,8 @@ namespace osu.Game.Graphics.Containers
     public class BeatSyncedContainer : Container
     {
         private int lastBeat;
-        private TimingControlPoint lastTimingPoint;
+        protected TimingControlPoint LastTimingPoint;
+        protected EffectControlPoint LastEffectPoint;
```
Comment (peppy) [inline osu.Game/Graphics/Containers/BeatSyncedContainer.cs:32]: If these are to be exposed like this, they need to be `private set;`

File: osu.Game/Screens/Menu/OsuLogo.cs [hunk 2]
```diff
@@ -378,6 +388,9 @@ protected override void OnMouseUp(MouseUpEvent e)
 
         protected override bool OnClick(ClickEvent e)
         {
+            //triangles.AccentColours = this.FindClosestParent<MainMenu>()!.GetBackgroundColours(2).Select(x => Graphics.Backgrounds.Triangles.InRange(x, 0)).ToArray();
+            //background.Colour = this.FindClosestParent<MainMenu>()!.GetBackgroundColours()[0];
+
```
Comment (peppy) [inline osu.Game/Screens/Menu/OsuLogo.cs:393]: Why is this removed?

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Game/Screens/Menu/OsuLogo.cs (hunk 2 above)
Comment (TacoGuyAT) [reply osu.Game/Screens/Menu/OsuLogo.cs:393]: Oh, my bad, parts left from old PR with multicolored triangles.

**Comment by peppy** [comment]
Pretty good direction. I've made some further tweaks which makes it feel even better for me. I also fixed the fact you were applying `Damp` on a single frame in `OnNewBeat` (which wouldn't really work as you wanted it to) by replacing with an immediate (but at a lower absolute value) velocity increase.

**Review by peppy — APPROVED** [review]
